internals/secrethub/audit.go

```diff
@@ -88,6 +88,11 @@ func (cmd *AuditCommand) run() error {
 				return err
 			}
 
+			isDir, err := client.Dirs().Exists(secretPath.Value())
```

Comment:
Do we want to continue if there is an error?

Reply:
I think so, yes. This check is here as an extra convenience: to return a more descriptive error than secret not found. 

When for some reason it is not possible to retrieve whether this is the case, I think it's better to continue in a degraded mode than always returning an error.
This would mean a less descriptive error when a directory is being tried to audit, but expected functionality when a secret is being audited (instead of the error return).